### ISSUE TITLE
How to use the darker background?

### ISSUE THREAD

**Issue by genesis-algorithms** [dracula/vim#208]
<img width="1234" alt="screenshot" src="https://user-images.githubusercontent.com/5767281/92585268-68c8d180-f29d-11ea-9158-28a2204b4ab4.png">

If I'm not wrong there are two background-colors for this theme (as you can see in the picture above).

I've even checked the files on this repo and there are indeed two background colors:

```
// vim/autoload/dracula.vim
let g:dracula#palette.bgdark    = ['#21222C', 235]
let g:dracula#palette.bgdarker  = ['#191A21', 234]
```

How can I use the darker one on my editor? Think I can directly change the color on my machine but there should be more elegant way.

Note: I'm pretty new with Nvim sorry if this question unnecessarily trivial.

**Comment by benknoble** [comment]
What you're seeing in the screenshot is vim in a terminal and MacVim.

MacVim is using gui colors and thus using the theme's background. Terminal uses fallback colors (256, in this case) when `termguicolors` isn't set (Terminal.app doesn't support truecolor). I happen to already have my Terminal set to a dracula palette as well, and I often turn off the dracula background in Terminal (see `g:dracula_colorterm`, I think), so it's possible that screenshot has my Terminal background, further confusing the matter. 

The short version is—there are slight differences between the terminal colors and gui colors.

---

If you really want a different background, you should be able to do something like

```vim
augroup my_dracula
  autocmd!
  autocmd ColorScheme dracula highlight Normal ...
augroup END
```

**Comment by genesis-algorithms** [comment]
Oooh, lol... Never thought about that, sorry :D. My `termguicolor` is already set on.

I would like to ask one more think. Right now when my NERDTree is open there this strangle padding with darker color around the separator "line":
<img width="1440" alt="Screen Shot 2020-09-09 at 22 55 36" src="https://user-images.githubusercontent.com/5767281/92647443-8e32fb00-f2f0-11ea-8b8d-8b7fe84bd799.png">

With tweaking these settings:
<img width="1440" alt="Screen Shot 2020-09-09 at 22 51 49" src="https://user-images.githubusercontent.com/5767281/92647496-a4d95200-f2f0-11ea-90e0-1473a2313b72.png">

I've managed to make it look "clean" like the following:
<img width="1440" alt="Screen Shot 2020-09-09 at 21 25 42" src="https://user-images.githubusercontent.com/5767281/92647536-b3276e00-f2f0-11ea-90fb-3b84e038c248.png">

But I wonder if there another way of achieving this?

**Comment by benknoble** [comment]
You could use the same autocmd approach with the `DraculaBoundary` group (which is want `VertSplit`, `Folded`, etc. link to). Note that it will affect other portions of the UI.

Alternately, you could set a custom highlight on `VertSplit`. But doing so breaks the link, so you have to manually override all the options you want.
